lib/Extension/LanguageServerEvaluatableExpression/LanguageServerEvaluatableExpressionExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerEvaluatableExpression;
+
+use Phpactor\Container\Container;
+use Phpactor\Container\ContainerBuilder;
+use Phpactor\Extension\WorseReflection\WorseReflectionExtension;
+use Phpactor\Extension\LanguageServer\LanguageServerExtension;
+use Phpactor\Extension\LanguageServerEvaluatableExpression\Handler\EvaluatableExpressionHandler;
+use Phpactor\Container\Extension;
+use Phpactor\MapResolver\Resolver;
+
+class LanguageServerEvaluatableExpressionExtension implements Extension
+{
+    public function load(ContainerBuilder $container): void
+    {
+        $container->register('language_server_completion.handler.evaluatable_expression', function (Container $container) {
+            return new EvaluatableExpressionHandler(
+                $container->get(LanguageServerExtension::SERVICE_SESSION_WORKSPACE),
```

Comment:
you can do `$container->expect(LanguageServerExtension::SERVICE_SESSION_WORKSPACE, Workspace::class)` to make phpstan happy.

Reply:
ok